apps/block_scout_web/test/block_scout_web/controllers/api/rpc/reward_controller_test.exs

```diff
@@ -469,88 +383,55 @@ defmodule BlockScoutWeb.API.RPC.RewardControllerTest do
       assert :ok = ExJsonSchema.Validator.validate(schema, response)
     end
 
-    test "with valid validator address", %{conn: conn} do
-      {validator_address_1_hash, group_address_1_hash, block_2_hash, block_3_hash} =
-        SetupValidatorAndGroupRewardsTest.setup()
-
-      expected_result = %{
-        "rewards" => [
-          %{
-            "amount" => "100000",
-            "date" => "2022-01-03T17:42:43.162804Z",
-            "blockNumber" => "10730880",
-            "blockHash" => to_string(block_2_hash),
-            "epochNumber" => "621",
-            "group" => to_string(group_address_1_hash)
-          },
-          %{
-            "amount" => "200000",
-            "date" => "2022-01-04T17:42:43.162804Z",
-            "blockNumber" => "10748160",
-            "blockHash" => to_string(block_3_hash),
-            "epochNumber" => "622",
-            "group" => to_string(group_address_1_hash)
-          }
-        ],
-        "totalRewardCelo" => "300000",
-        "from" => "2022-01-03 00:00:00.000000Z",
-        "to" => "2022-01-06 00:00:00.000000Z",
-        "account" => to_string(validator_address_1_hash)
-      }
-
-      response =
-        conn
-        |> get("/api", %{
-          "module" => "reward",
-          "action" => "getvalidatorrewards",
-          "validatorAddress" => to_string(validator_address_1_hash),
-          "from" => "2022-01-03T00:00:00.000000Z",
-          "to" => "2022-01-06T00:00:00.000000Z"
-        })
-        |> json_response(200)
-
-      assert response["result"] == expected_result
-      assert response["status"] == "1"
-      assert response["message"] == "OK"
-      schema = generic_rewards_schema()
-      assert :ok = ExJsonSchema.Validator.validate(schema, response)
-    end
-
     test "with valid validator address list", %{conn: conn} do
```

Comment:
A test that covers the case when you have multiple validators but you're fetching rewards for one is missing for completeness.